event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.CaseUpdate.json

```diff
@@ -15,5 +16,7 @@
   },
   "sampleSensitive": {
     "phoneNumber": "REDACTED"
-  }
+  },
+  "createdAt": "1970-01-01T00:00:00.000Z",
+  "lastUpdatedAt": "1970-01-01T00:00:00.000Z"
```

Comment:
prefer more recent date / time . Please check it comes out correctly with JSON conversion, perhaps with unit test.

Reply:
Tested locally and added small test to confirm